diabetes_regression/register/register_model.py

```diff
@@ -98,32 +90,36 @@ def main():
     model_file = os.path.join(model_path, model_name)
     model = joblib.load(model_file)
     model_mse = run.parent.get_metrics()["mse"]
+    parent_tags = run.parent.get_tags()
+    try:
+        build_id = parent_tags["BuildId"]
+    except KeyError:
+        build_id = None
```

Comment:
I would log something saying we didn't see a build ID, so that we can more easily debug if we expected to see one. Same for build URI.